### PR TITLE
Fix case detection for another type of indirect jump table

### PR DESCRIPTION
**Pull request by GustavoLCR**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

### CODE REVIEW

File: libr/anal/fcn.c [hunk 1]
```diff
@@ -228,7 +228,7 @@ static bool is_delta_pointer_table(RAnal *anal, RAnalFunction *fcn, ut64 addr, u
 		if (o_reg_dst && reg_src && o_reg_dst->offset == reg_src->offset && omov_aop.disp != UT64_MAX) {
 			// Special case for indirection
 			// lea reg1, [base_addr]
-			// mov reg2, (sz)word [reg1 + tbl_off*sz + casetbl_loc_off]
+			// movzx reg2, byte [reg1 + tbl_off + casetbl_loc_off]
```
Comment (XVilka) [inline libr/anal/fcn.c:231]: Why this change? Since you add a *new* type of indirect jump table. Why change the old comment?

File: libr/anal/fcn.c [hunk 2]
```diff
@@ -1116,7 +1116,26 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut64 len, int
 				if (op.ptr != UT64_MAX && op.ireg) { // direct jump
 					ut64 table_size, default_case;
 					if (try_get_jmptbl_info (anal, fcn, op.addr, bb, &table_size, &default_case)) {
-						ret = try_walkthrough_jmptbl (anal, fcn, bb, depth, op.addr, op.ptr, op.ptr, anal->bits >> 3, table_size, default_case, ret);
+						bool case_table = false;
```
Comment (XVilka) [inline libr/anal/fcn.c:1119]: We probaly should split that function in a few smaller ones in the future.

File: libr/anal/fcn.c (hunk 1 above)
Comment (GustavoLCR) [reply libr/anal/fcn.c:231]: I noticed the comment was wrong, so I fixed it
